server/routerlicious/packages/services/src/redisSessionManager.ts

```diff
@@ -47,6 +47,22 @@ interface IShortCollaborationSession {
 	};
 }
 
+/**
+ * Options for {@link RedisCollaborationSessionManager} that tune specific behaviors.
+ */
+interface IRedisCollaborationSessionManagerOptions {
+	/**
+	 * Maximum number of sessions to scan at once when calling {@link RedisCollaborationSessionManager.getAllSessions}.
+	 * The expected return size of a given session is less than 200 bytes, including the key.
+	 * @defaultValue 800 - Intended to keep each scan result under 200KB (which would be ~1000 sessions).
+	 */
+	maxScanBatchSize: number;
+}
+
+const defaultRedisCollaborationSessionManagerOptions: IRedisCollaborationSessionManagerOptions = {
+	maxScanBatchSize: 800,
```

Comment:
Should we make this configurable?

Reply:
Probably, but we instantiate our own `RedisCollaborationSessionManager` in AFR in the cronjob we use instead of the OSS interval. That's where I'll plug in the configs, because it's not really relevant here.

I can plumb it through here as well for completeness.